compiler/test/dotty/tools/utils.scala

```diff
@@ -74,31 +74,35 @@ def toolArgsFor(files: List[JPath], charset: Charset = UTF_8): ToolArgs =
     }
   }
 
-def toolArgsFor(tool: ToolName)(lines: List[String]): List[String] =
-  toolArgsParse(lines).collectFirst { case (name, args) if tool eq ToolName.named(name) => CommandLineParser.tokenize(args) }.getOrElse(Nil)
+def toolArgsFor(tool: ToolName, filename: Option[String])(lines: List[String]): List[String] =
+  toolArgsParse(lines, filename).collectFirst { case (name, args) if tool eq ToolName.named(name) => CommandLineParser.tokenize(args) }.getOrElse(Nil)
 
 // scalac: arg1 arg2, with alternative opening, optional space, alt names, text that is not */ up to end.
```

Comment:
this looks like it should be updated as well?

Reply:
thanks, have done